src/python/dxpy/bindings/dxproject.py

```diff
@@ -282,8 +282,9 @@ class DXProject(DXContainer):
     _class = "project"
 
     def new(self, name, summary=None, description=None, protected=None,
-            restricted=None, download_restricted=None, contains_phi=None, tags=None,
-            properties=None, bill_to=None, **kwargs):
+            restricted=None, download_restricted=None, contains_phi=None, 
+            tags=None, properties=None, bill_to=None, database_ui_view_only=None,
```

Comment:
I added the new `database_ui_view_only` flag here so that it can be set with our `dxpy` bindings.  

Something I noticed is that while `contains_phi` can be passed here, we don't seem to be doing this for `update` ([link](https://github.com/dnanexus/dx-toolkit/blob/95bfc3ed51383094eb597394ed65f24ea4fd6646/src/python/dxpy/bindings/dxproject.py#L346-L348)).

Is that intentional?  Should I include the `database_ui_view_only` flag in the `update` function?

Reply:
Yes, please add `database_ui_view_only` also to update, similar to `restricted` and `download_restrcited`. I think omitting `contains_phi` was intentional because once project is set to contain PHI, it can never be updated to not contain them.